src/features/user_profile/components/SettingsForm/ReferenceAreaInfo/ReferenceAreaInfo.tsx

```
@@ -112,10 +109,7 @@ export function ReferenceAreaInfo() {
                 {i18n.t('profile.reference_area.accessing_location_error')}
               </span>
             ) : (
-              <button
-                className={clsx(s.link, s.userLocation)}
-                onClick={onSelectCurrentLocation}
-              >
+              <button className={s.link} onClick={onSelectCurrentLocation}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)